### PR TITLE
Options in redux middlewares

### PR DESCRIPTION
**Pull request by priyansh1161**
This introduces a convenient way to provide options to redux middlewares.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,14 +60,24 @@ check [server](#server) config for more details.
...
+| stateFilter | No | Function | `() => null` | function to filter state [See more](#statefilter)
+| level | No | String | `info` | syslog level
 
 **actionFilters** and **stateFilters** can be applied to filter out what is logged. [see more](#filters)
 
 ### CrashReporter
 
-`crashReporter(logger [,stateFilter, level])` is a redux middleware which reports any crash, uncaught errors, uncaught promises to WebCLS
-as `error` level logs.
+`crashReporter(logger [, options])` is a redux middleware which reports any crash, uncaught errors, uncaught promises to WebCLS.
```
Comment (nimishsinghal) [inline README.md]: minor knit pick `any crash, uncaught errors & uncaught promises`

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "web-central-logging",
-  "version": "0.2.0",
+  "version": "0.3.1",
```
Comment (nimishsinghal) [inline package.json]: why 3.1? and not 3.0?

**Comment by nimishsinghal** [comment]
Approved

**Comment by dash1291** [comment]
Approved

**Pull request merged**